ramls/dataExport.raml

```diff
@@ -11,15 +11,15 @@ documentation:
 
 types:
   errors: !include raml-util/schemas/errors.schema
+  fileDefinition: !include schemas/fileDefinition.json
   exportRequest: !include schemas/exportRequest.json
 
 traits:
   validate: !include raml-util/traits/validation.raml
-  language: !include raml-util/traits/language.raml
 
 /data-export:
   /export:
-    displayName: Data export service
+    displayName: Export manager service
```

Comment:
I think this was the description Magda was mentioning last time that we need to be more elaborate. Can we confirm with her once